web/bower.json

```diff
@@ -31,6 +31,7 @@
     "Leaflet.utfgrid": "danzel/Leaflet.utfgrid",
     "d3": "3.5.5",
     "d3-tip": "^0.6.7",
+    "world-calendars": "azavea/calendars#feature/islamic-picker-ar",
```

Comment:
use project specific branches, instead of language, so we can support multiple languages on a single project